Source/Paths/SvgPathBuilder.cs

```diff
@@ -32,8 +32,13 @@ public static SvgPathSegmentList Parse(string path)
 
             try
             {
-                foreach (var commandSet in SplitCommands(path.TrimEnd(null)))
-                    CreatePathSegment(commandSet[0], segments, new CoordinateParser(commandSet.Trim()));
+                var pathTrimmed = path.AsSpan().TrimEnd();
+                var commands = SplitCommands(ref pathTrimmed);
+                foreach (var commandSet in commands)
+                {
+                    var commandSetTrimmed = pathTrimmed.Slice(commandSet.Start, commandSet.Length).Trim();
+                    CreatePathSegment(commandSetTrimmed[0], segments, new CoordinateParser(ref commandSetTrimmed), ref commandSetTrimmed);

```

Comment:
new CoordinateParser(ref commandSetTrimmed),

I think the Parser could be kept in a variable var parser = new CoordinateParser();
and then in the iteration this could be called.
parser.Init(ref commmandSetTrimmed);

In the Init method the all would be initialized like in the Constructor

Reply:
Add Init method to CoordinateParser